### ISSUE TITLE
Add ingress example

### ISSUE THREAD

**Issue by poikilotherm** [IQSS/dataverse-kubernetes#97]
Copied from #95 by @cledevedec:
> I also had to add an `ingress` like I was using **traefik** as a reverse proxy for my cluster.

We should provide an example deployment for traefik in case someone needs it.

**Comment by cledevedec** [comment]
Here is how you can use traefik for your ingress  
```
apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  annotations:
    certmanager.k8s.io/cluster-issuer: letsencrypt
    kubernetes.io/ingress.class: traefik
    traefik.ingress.kubernetes.io/redirect-entry-point: https
  labels:
    environment: <your-env>
    k8s-app: dataverse
    trigram: k8s
  name: dataverse
  namespace: dataverse-test
spec:
  tls:
  - hosts:
    - <@host>
    secretName: dataverse-certs
  rules:
  - host: <@host>
    http:
      paths:
      - backend:
          serviceName: dataverse
          servicePort: 8080
        path: /
status:
  loadBalancer: {}
```

**Comment by poikilotherm** [comment]
For #91 a few examples have been added that are actually used in the demos. More to come, especially for production usage. This is related to #92 and should be attacked in the same breath.

**Comment by poikilotherm** [comment]
Those first examples have been included now in the new guide and #92 has been solved today.
We should add more on this for AWS #95, Azure #108 etc, but closing this for now.
